app-shells/mksh/mksh-57-r1.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit eutils toolchain-funcs
+
+if [[ $PV = 9999 ]]; then
+	inherit cvs
+	ECVS_SERVER="anoncvs.mirbsd.org:/cvs"
+	ECVS_MODULE="mksh"
+	ECVS_USER="_anoncvs"
+	ECVS_AUTH="ext"
+	KEYWORDS=""
+else
+	SRC_URI="http://www.mirbsd.org/MirOS/dist/mir/mksh/${PN}-R${PV}.tgz"
+	KEYWORDS="~amd64 ~ppc ~x86 ~amd64-linux ~x86-linux"
+fi
+
+DESCRIPTION="MirBSD Korn Shell"
+HOMEPAGE="http://mirbsd.de/mksh"
+LICENSE="BSD"
+SLOT="0"
+IUSE="static"
+DEPEND="static? ( dev-libs/klibc )"
+RDEPEND=""
+S="${WORKDIR}/${PN}"
+
+src_compile() {
+	tc-export CC
+	# we want to build static with klibc
+	if use static; then export CC="/usr/bin/klcc" export LDSTATIC="-static"; fi
+	export CPPFLAGS="${CPPFLAGS} -DMKSH_DEFAULT_PROFILEDIR=\\\"${EPREFIX}/etc\\\""
```

Comment:
Oh, it would be better to use `append-cxxflags` from `flag-o-matic` instead, right?

Reply:
No, there's really no difference and no preference either way.